clusterloader2/pkg/measurement/common/api_responsiveness.go

```diff
@@ -78,7 +78,7 @@ func (*apiResponsivenessMeasurement) Execute(config *measurement.MeasurementConf
 	case "gather":
 		// TODO(krzysied): Implement new method of collecting latency metrics.
 		// New method is defined here: https://github.com/kubernetes/community/blob/master/sig-scalability/slos/slos.md#steady-state-slisslos.
-		nodeCount, err := util.GetInt(config.Params, "nodeCount")
+		nodeCount, err := util.GetIntOrDefault(config.Params, "nodeCount", config.ClusterConfig.Nodes)
```

Comment:
Weren't we gonna compute this by api call instead of taking it as parameter?

Reply:
Computing parameters will be next step. I will create PR after this one is merged.

Reply:
Then please at least add a TODO.